rust/operator-binary/src/controller.rs

```diff
@@ -616,12 +617,27 @@ fn build_rolegroup_config_map(
                 // The size of one log file
                 dynamic_resolved_config.insert(
                     LOG_MAX_SIZE.to_string(),
-                    Some(format!("{TRINO_LOG_FILE_SIZE_IN_MIB}MB")),
+                    Some(format!(
+                        // Trino uses the unit "MB" for MiB.
+                        "{}MB",
+                        MAX_TRINO_LOG_FILES_SIZE
+                            .scale_to(BinaryMultiple::Mebi)
+                            .div(LOG_FILE_COUNT as f32)
+                            .ceil()
+                            .value,
+                    )),
                 );
                 // The maximum size of all logfiles combined
                 dynamic_resolved_config.insert(
                     LOG_MAX_TOTAL_SIZE.to_string(),
-                    Some(format!("{TRINO_LOG_FILE_TOTAL_SIZE_IN_MIB}MB")),
+                    Some(format!(
+                        // Trino uses the unit "MB" for MiB.
+                        "{}MB",
+                        MAX_TRINO_LOG_FILES_SIZE
+                            .scale_to(BinaryMultiple::Mebi)
+                            .ceil()
```

Comment:
ceil or floor? :thinking:

Reply:
Let's assume that the developer sets a reasonable value of 10 MiB:
LOG_MAX_SIZE is 5MB
LOG_MAX_TOTAL_SIZE is 10MB
⇒ okay

If the developer sets an odd value like 2.2 MiB:
LOG_MAX_SIZE is 2MB
LOG_MAX_TOTAL_SIZE is 3MB
⇒ okay, I guess

If the developer sets a tiny value like 0.2 MiB:
LOG_MAX_SIZE is 1MB
LOG_MAX_TOTAL_SIZE is 1MB
⇒ not okay (also with floor)